### PR TITLE
[SYCLomatic] Move Math APIs together in APINames.inc.

### PR DESCRIPTION
**Pull request by tangjj11**
Signed-off-by: Tang, Jiajun jiajun.tang@intel.com

### CODE REVIEW

**Review by wenhuiNi — APPROVED** [review]

File: clang/lib/DPCT/APINames.inc [hunk 1]
```diff
@@ -570,13 +631,109 @@ ENTRY(h2sin, h2sin, true, NO_FLAG, P4, "Successful", "NA")
 ENTRY(h2sqrt, h2sqrt, true, NO_FLAG, P4, "Successful", "NA")
 ENTRY(h2trunc, h2trunc, true, NO_FLAG, P4, "Successful", "NA")
 
+// 16 Bfloat16 Arithmetic Functions are overload with other ENTRY
+
+// 17 Bfloat162 Arithmetic Functions are overload with other ENTRY
+
+// 18 Bfloat16 Comparison Functions are overload with other ENTRY
+
+// 41 Bfloat162 Comparison Functions are overload with other ENTRY
```
Comment (wenhuiNi) [inline clang/lib/DPCT/APINames.inc]: if this is useless code, please remove it

**Review by wenhuiNi — CHANGES_REQUESTED** [review]

File: clang/lib/DPCT/APINames.inc (hunk 1 above)
Comment (tangjj11) [reply clang/lib/DPCT/APINames.inc]: I'm not sure whether should we add these comments when there are overload entry. I will confirm it.

Comment (tangjj11) [reply clang/lib/DPCT/APINames.inc]: I have optimized the description here.

**Review by wenhuiNi — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**